specification/sql/resource-manager/Microsoft.Sql/preview/2023-02-01-preview/LocationCapabilities.json

```diff
@@ -693,6 +693,11 @@
           "type": "string",
           "readOnly": true
         },
+        "isGeneralPurposeV2": {
```

Comment:
Thanks for the input.

I do understand your reasoning, and I agree that it sounds like a better design in general.

However:
- this is just one part of the API changes. The other part is the managed instance API, which uses the same flag. Both DSMain and swagger changes are merged and afaik it's already undergoing ARM deployment together with two other changesets - all three are parts of features that are on a critical timeline.
- regarding this API (location capabilities), the DS Main changes are already done
- I did not get any such input on the previous reviews (DSMain for MI, DSMain for LocCap, swagger for MI)
- I also did not get any such input on the design review
- This flag is set to be deprecated at some point in the next 2-3 years when everything is V2 (it will default to true) - and it will never have more than two values

Given all the above, I would proceed with this design.